introduction.md

```diff
@@ -0,0 +1,3 @@
+Look! I am just playing with CodeSchool introduction to github course. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.